rendering.fbs

```diff
@@ -19,9 +19,28 @@ table Color {
   b:ubyte = 0;
 }
 
+table BallAnchor {
+  local:Vector3 (required);
+}
+
+table CarAnchor {
+  index:uint;
```

Comment:
why is this not required?

Reply:
i just pushed a commit that messed this up - what part are you talking about?

Reply:
the code you highlight just doesn't point to anything
![image](https://github.com/RLBot/flatbuffers-schema/assets/35614515/9af7ce1e-feaa-4b67-854f-69defd572a47)

Reply:
unless you do mean `index`? all primitive types like `uint` are required by default (and actually can't be optional at all)

Reply:
lgtm then